enmanuelPersonal/LaPaz-Back-End
# Adding models for user records

# [Adding models for user records](https://trello.com/c/kC7A2rMm/12-as-a-developer-i-would-like-to-create-the-model-and-handler-for-a-user-and-be-able-to-generate-tokens)

##  Description
This PR Add models, handlers and tokens for user registries.


## Testing Steps
- Start the project.
- To generate data in the database execute `npm run seeders`.
- With a request service like `PostMan`. Send requests to `POST:http://localhost:8080/user/add` . 
  - You can use this json: {
  "usuario": "EnmanuelEstrella22",
    "password": "12345678",
    "idTipoUsuario": "2c961280-b2b5-4188-aa01-8e47a524c362",
    "nombre" : "Enmanuel",
    "nacimiento" : "12/22/1999",
    "telefonos" : [{"telefono":"8096125752", "tipo":"0083ccc0-8aa2-4367-b16e-cdebbc3736e8"}, {"telefono":"8492777475", "tipo":"b079fbf7-09f4-4a80-b0c9-3750114a74ae"}],
    "correos" : ["e@gmail.com", "a@gmail.com"],
    "direcciones" : [{"pais": "Republica Dominicana", "region":"Norte", "ciudad": "Santiago", "municipio":"Punal", "sector": "Laguna Prieta", "calle":"Los estrellas", "casa":"16", "referencia": "frente a la banca"}]
}
- Verify that it has been added correctly.
